app/javascript/dashboard/routes/dashboard/settings/inbox/channels/Whatsapp.vue

```diff
@@ -67,29 +57,31 @@ const selectProvider = providerValue => {
   });
 };
 
-const shouldShowEmbeddedSignup = provider => {
-  // Check if the feature is enabled for the account
-  if (!isWhatsappEmbeddedSignupEnabled.value) {
-    return false;
-  }
-
+const shouldShowCloudWhatsapp = provider => {
   return (
-    (provider === PROVIDER_TYPES.WHATSAPP && hasWhatsappAppId.value) ||
-    provider === PROVIDER_TYPES.WHATSAPP_EMBEDDED
+    provider === PROVIDER_TYPES.WHATSAPP_MANUAL ||
+    (provider === PROVIDER_TYPES.WHATSAPP && !hasWhatsappAppId.value)
   );
 };
 
-const shouldShowCloudWhatsapp = provider => {
-  // If embedded signup feature is enabled and app ID is configured, don't show cloud whatsapp
-  if (isWhatsappEmbeddedSignupEnabled.value && hasWhatsappAppId.value) {
-    return false;
-  }
-
-  // Show cloud whatsapp when:
-  // 1. Provider is whatsapp AND
-  // 2. Either no app ID is configured OR embedded signup feature is disabled
-  return provider === PROVIDER_TYPES.WHATSAPP;
+const handleManualLinkClick = event => {
+  event.preventDefault();
+  selectProvider(PROVIDER_TYPES.WHATSAPP_MANUAL);
 };
+
+const manualFallbackText = computed(() => {
```

Comment:
This is not a good approach. Using `v-html` for string replacement can introduce security issues.

<img width="443" height="276" alt="Image" src="https://github.com/user-attachments/assets/af106ee6-0691-4468-878d-becc29a86448" />